src/diagrams/flowchart/parser/flow.jison

```diff
@@ -271,7 +271,7 @@ document
 	{ $$ = [];}
 	| document line
 	{
-	    if($2 !== []){
+	    if(!Array.isArray($2) || $2.length > 0){
```

Comment:
@knsv esbuild was throwing a warning that comparison `!== []` will always be true.